userguide/sharing.rst

```diff
@@ -1092,16 +1092,17 @@ settings:
   in
   :menuselection:`Services --> SMB`.
 
-* Be careful about unchecking the
-  :guilabel:`Browsable to Network Clients` box. When this box is
-  checked (the default), other users will see the names of every share
-  that exists using Windows Explorer, but they will receive a
-  permissions denied error message if they try to access someone
-  else's share. If this box is unchecked, even the owner of the share
-  will not see it or be able to create a drive mapping for the share
-  in Windows Explorer. However, they can still access the share from
-  the command line. Unchecking this option provides limited security
-  and is not a substitute for proper permissions and password control.
+* When the :guilabel:`Browsable to Network Clients` box is checked (the 
+  default), the share will be visible in the server's browse list. If the share 
+  is is flagged as a :guilabel:`home share`, unchecking this box will hide the 
+  [homes] share so that only the dynamically generated home share with the
+  authenticated user's name will be visible (in contrast with [homes]
+  and [<username>] both being visible). 
+
+  The presence of a share in a user's browse list does not grant the user  
+  read or write permissions on the share. Users are still able to directly 
+  access and map shares that are not visible in the browse list by using 
+  the share's UNC path. Hence, this option is of limited security utility.
```

Comment:
"This option provides no real security because shares that are not visible in the File Explorer (or whatever) can still be accessed with a *UNC* path."

Not sure about markup, but I think that's what this is trying to say.